testcases/MachineConfig.py

```diff
@@ -56,16 +55,42 @@ def setUp(self):
             self.system_name = conf.args.system_name
             self.cv_HMC = self.cv_SYSTEM.hmc
             self.lpar_prof = conf.args.lpar_prof
+            self.secureboot_valid_states = ['on', 'off']
         else:
             self.skipTest("Functionality is supported only on LPAR")
 
+    def check_secureboot_state(self):
```

Comment:
please move this to optestutils

Reply:
Renamed to validate_secureboot_parameter() and moved to OpTestUtil.py